internal/goruntime/memory.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
How about put them in util? seems the internal pkg is related to go, and it's not a internal pkg of volcao, so it's a little wired to put them in the root directory.

Reply:
Thanks @Monokaix . Using Internal pkg ensure that certain parts of their codebase are "shielded" from external use.
But It is also possible to move it under pkg.